packages/account/src/Components/forms/personal-details-form.jsx

```diff
@@ -109,473 +103,587 @@ const PersonalDetailsForm = ({
+                    {'phone' in values && (
+                        <PhoneField
+                            value={values.phone}
+                            editable_fields={editable_fields}
+                            has_real_account={has_real_account}
+                            required
+                        />
                     )}
-                </fieldset>
-            </FormBodySection>
-        </div>
+                    <React.Fragment>
+                        {'place_of_birth' in values && (
+                            <PlaceOfBirthField
+                                handleChange={handleChange}
+                                setFieldValue={setFieldValue}
+                                editable_fields={editable_fields}
+                                residence_list={residence_list}
+                                required
+                            />
+                        )}
+                        {'tax_residence' in values && (
+                            <TaxResidenceField
+                                setFieldValue={setFieldValue}
+                                editable_fields={editable_fields}
+                                residence_list={residence_list}
+                                setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                                setIsTinPopoverOpen={setIsTinPopoverOpen}
+                                is_tax_residence_popover_open={is_tax_residence_popover_open}
+                                required
+                            />
+                        )}
+                        {'tax_identification_number' in values && (
+                            <TaxIdentificationNumberField
+                                warning_items={warning_items}
+                                is_tin_popover_open={is_tin_popover_open}
+                                setIsTinPopoverOpen={setIsTinPopoverOpen}
+                                setIsTaxResidencePopoverOpen={setIsTaxResidencePopoverOpen}
+                                editable_fields={editable_fields}
+                                required
+                            />
+                        )}
+                        {'account_opening_reason' in values && (
+                            <AccountOpeningReasonField
+                                noHeader
+                                account_opening_reason_list={account_opening_reason_list}
+                                setFieldValue={setFieldValue}
+                                editable_fields={editable_fields}
+                                required
+                            />
+                        )}
+                    </React.Fragment>
+                </div>
+            )}
+        </React.Fragment>
     );
 };
 
 export default PersonalDetailsForm;
+
+const PhoneField = ({ value, editable_fields, has_real_account, required }) => (
```

Comment:
@shahzaib-deriv I don't think adding more lines of code to this file is a good idea, can we separate the fields into other smaller components? 
@likhith-deriv @yashim-deriv what do you think?

Reply:
We will do the refactoring in a separate card.